examples/nek5000/bp1.usr

```diff
@@ -825,23 +825,21 @@ c     Create ceed basis for mesh and computation
       p=nx1
       q=p+1
       ncomp=1
-      call ceedbasiscreatetensorh1lagrange(ceed,ndim,ndim,q,q,
```

Comment:
It doesn't matter in this case, as you aren't creating or using the coordinates, but basisx should have `ndim` fields, i.e. x,y,z has 3 fields.

Reply:
Can you explain further? I don't quite understand.

Reply:
In PETSc `bp1.c` lines 320-334 we set up the coordinates vector. It requires 3 components, one for the x coordinate, one for the y coordinate, and one for the z coordinate.

The `ncomp` field in this basis is not ever used, as this example never actually creates a coordinate vector in libCEED and this basis never acts on a coordinate vector, so this field does not have an effect, but technically the coordinates would have 3 components in 3D.

I set `ncomp` to `ndim` when I converted this example in case a future user uses this example as a baseline to make a libCEED Fortran code on a more complex mesh. In that (future) case, that user would need to set `ncomp` to `ndim`, so I modeled that for them.

I'm not sure if I explained it better, so please let me know if I can be more clear!